examples/dreambooth/train_dreambooth_lora_flux.py

```diff
@@ -951,9 +963,10 @@ def encode_prompt(
     text_encoders,
     tokenizers,
     prompt: str,
-    max_sequence_length,
+    max_sequence_length=512,
```

Comment:
Let's not enforce a default value here and always require the user to pass it from the CLI? We can make the CLI arg default though.